CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - `go foo()` has been replaced with `spawn foo()` (launches an OS thread, `go` will be used for
   upcoming coroutines instead).
 - vfmt now supports `// vfmt off` and `// vfmt on` for turning off the formatting locally for *short* snippets of code. Useful for keeping your carefully arranged matrices in tact.
```

Comment:
I know this isn't part of your stuff, but it would be nice to fix it while you're adding to the file anyway...
```suggestion
- vfmt now supports `// vfmt off` and `// vfmt on` for turning off the formatting locally for *short* snippets of code. Useful for keeping your carefully arranged matrices intact.
```

Reply:
ok

Reply:
done in 69f90b6